en/mulesoft/modules/ROOT/pages/ms_composer_slack_reference.adoc

```diff
@@ -53,12 +53,47 @@ Post a new message to a specific private or public channel that you select. To p
 * *Bot Name*: By default, the sender of Slack messages generated by an action is *MuleSoft Composer*. You can supply a different name for the messages sent by this action.
 * *Bot Icon URL*: By default, the sender of Slack messages generated by an action is *MuleSoft Composer*, and includes the MuleSoft logo in icon form. You can supply a different icon for the messages sent by this action. Specify an icon URL whose access is not restricted.
 
-Get user by email address::
+Get user by email::
 
 Find a Slack user by entering their email address into the *Email* field.
 +
 If you want to fetch the user's ID dynamically, you can insert this action, which fetches the Slack user by an email address from another system. Use the *ID* data pill returned by this action to specify the user who should receive the message.
 
+Add users to channel::
+
+Invites users to a channel. This action requires a new connection. Provide a value for the following fields:
++
+* *Channel Name*: A channel in the Slack workspace that you select from a drop-down list of the existing channels.
+* *User IDs*: IDs of the users to add to a channel.
+
+Create channel::
+
+Creates a public or private channel-based conversation. This action requires a new connection. Provide a value for the following fields:
++
+* *Name*: The name of the channel to create.
+* *Is Private*: Determines whether the channel to create is private or public.
+* *Team Id*: The encoded team ID to create the channel in, which is required if the org token is used.
```

Comment:
Suggest: The encoded ID of the team in which to create the channel. This ID is required...